lib/tasks/cards.rake

```diff
@@ -310,8 +310,8 @@ namespace :cards do
 
     # Collect each card pool's cycles
     card_pools.each do |p|
-      next if p['cycles'].nil?
-      p['cycles'].each do |s|
+      next if p['card_cycle_ids'].nil?
```

Comment:
Outside of this pr, can you use dot syntax for these accesses?

Reply:
sure.  i'll do a cleanup CL.  Is there anything gained aside from -3 characters per access?

Reply:
Preface: I don't exactly tenement so take this with a grain of salt.

That being said, I don't think so and it doesn't actually matter so you don't have to change over if you don't want to. It was something I had to do when the properties used dashes instead of underscores and now that they're underscores you can use dot syntax, but like javascript, they're equivalent so don't do extra work if you don't wanna.